packages/node/src/e2e/e2e-test.spec.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { it, describe } from "@jest/globals";
+import { Session } from "../Session";
+
+const OIDC_ISSUER = "https://broker.demo-ess.inrupt.com/";
+
+// This first test just saves the trouble of looking for a library failure when
+// the environment wasn't properly set.
+describe("Environment", () => {
+  it("contains the expected environment variables", () => {
+    expect(process.env.REFRESH_TOKEN).not.toBeUndefined();
+    expect(process.env.CLIENT_ID).not.toBeUndefined();
+    expect(process.env.CLIENT_SECRET).not.toBeUndefined();
+  });
+});
+
+describe("Authenticated fetch", () => {
```

Comment:
One other test that might be useful is to verify that, after logging in one session, a fetch from another session is still unauthenticated.

Reply:
Resolved in 9aa1fd7c8349f5d22f61481117f9a93d675471eb